### PR TITLE
D363: Add tool to convert file to base64

### PR DESCRIPTION
**Pull request by oferb**

### CODE REVIEW

File: common/Base64Tool.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ * limitations under the License.
+ */
+
+package com.google.startupos.common;
+
+import com.google.startupos.common.CommonModule;
+import dagger.Component;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import com.google.startupos.common.FileUtils;
```
Comment (val-fed) [inline common/Base64Tool.java]: unnecessary imports:
- import com.google.startupos.common.CommonModule;
- import com.google.startupos.common.FileUtils;

File: common/Base64Tool.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  private FileUtils fileUtils;
+
+  @Inject
+  Base64Tool(FileUtils fileUtils) {
+    this.fileUtils = fileUtils;
+  }
+
+  void run() throws IOException {
+    String fileContents = fileUtils.readFile(file.get());
```
Comment (val-fed) [inline common/Base64Tool.java]: Maybe s / fileContents / fileContent ?
Also `encodedContents` and `decodedContents`.

File: common/Base64Tool.java (hunk 1 above)
Comment (oferb) [reply common/Base64Tool.java]: Done

File: common/Base64Tool.java (hunk 2 above)
Comment (oferb) [reply common/Base64Tool.java]: Actually, contents sounds better. Here's a related read:
https://jakubmarian.com/content-vs-contents-in-english/

**Pull request merged**